app/experimenter/experiments/admin/nimbus.py

```diff
@@ -71,8 +71,15 @@ class NimbusExperimentAdmin(admin.ModelAdmin):
         NimbusBranchInlineAdmin,
         NimbusExperimentChangeLogInlineAdmin,
     )
-    list_display = ("name", "status", "application", "channel", "firefox_min_version")
-    list_filter = ("status", "application")
+    list_display = (
+        "name",
+        "status",
+        "publish_status",
+        "application",
+        "channel",
+        "firefox_min_version",
+    )
+    list_filter = ("status", "publish_status", "application")
```

Comment:
Yeah I'm guilty of sneaking admin changes like this into other PRs too.  Only problem with it is sometimes you end up having to back a PR out for some reason and then forget 'wait I thought I added this to the admin....'

Reply:
I could split that out in to a separate PR if need be. It turned out useful to verify what this PR was doing along with [the not-yet-PR'd UI work](https://github.com/mozilla/experimenter/compare/4800-request-review-ui) I've got in progress

Reply:
Yeah totally we also need to add a couple other things to the admin as well.  Actually yeah if you wanna break it out I can tack on the other ones I need too.

Reply:
Or not XD